sdcm/prometheus.py

```diff
@@ -181,7 +186,29 @@ def _publish_end_of_alerts(self, alerts: dict):
             if not alert.get('endsAt', None):
                 alert['endsAt'] = time.strftime("%Y-%m-%dT%H:%M:%S.0Z", time.gmtime())
```

Comment:
@dkropachev Hey Dmitry, do you happen to remember why we need to explicitly set the `endsAt` attribute for alerts in `PrometheusAlertManagerListener`? Shouldn't AlertManager do it? My concern here is that we might be artificailly marking some events as being shorter than they were in reality.

Reply:
This function get's events that are not returned by alertmanager anymore, so when they missed and we catch it and set end date to them if it is missing